Eatery/Onboarding/Controllers/OnboardingLoginViewController.swift

```diff
@@ -0,0 +1,238 @@
+//
+//  OnboardingLoginViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/12/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import NVActivityIndicatorView
+import UIKit
+
+class OnboardingLoginViewController: OnboardingViewController {
+
+    private var loginStackView: UIStackView!
+
+    private let netidPromptLabel = UILabel()
+    private let netidTextField = UITextField()
+
+    private let passwordPromptLabel = UILabel()
+    private let passwordTextField = UITextField()
+
+    private let loginButton = UIButton()
+    private let skipButton = UIButton()
+    private let privacyStatementButton = UIButton(type: .system)
+
+    private let activityIndicator = NVActivityIndicatorView(frame: .zero, type: .circleStrokeSpin, color: .white)
+    
+    private let errorView = BRBLoginErrorView()
+
+    private let accountManager = BRBAccountManager()
+
+    private var isLoading: Bool = false {
+        didSet {
+            if isLoading {
+                activityIndicator.startAnimating()
+                loginButton.setTitle(nil, for: .normal)
+            } else {
+                activityIndicator.stopAnimating()
+                loginButton.setTitle("LOG IN", for: .normal)
+            }
+
+            stackView.isUserInteractionEnabled = !isLoading
+            loginButton.isEnabled = !isLoading
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        accountManager.delegate = self
+        accountManager.removeSavedLoginInfo()
+
+        setUpStackView()
+        setUpPrivacyStatementButton()
+        setUpErrorView()
+        setUpNetidViews()
+        setUpPasswordViews()
+        setUpSkipButton()
+    }
+
+    private func setUpStackView() {
+        loginStackView = UIStackView()
+        loginStackView.axis = .vertical
+        loginStackView.distribution = .fill
+        loginStackView.spacing = 10
+        stackView.insertArrangedSubview(loginStackView, at: 2)
```

Comment:
Instead of inserting an arranged subview at index 2, it may be worthwhile to have this functionality be attached to the `OnboardingViewController`. 

This way, we could have `stackView` be private in the `OnboardingViewController`, as well as remove this peculiar insertion into a stack view at index 2.